ship.d/run

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+set -ev
+
+CMD="python /app/start.py"
+exec su acacia -c "$CMD"
```

Comment:
this should also pass all other command line params along to $CMD.  The calling script is passing them into here (ship:33)